common/thorhelper/thorsoapcall.cpp

```diff
@@ -1919,6 +1919,7 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
                 StringBuffer s;
                 master->logctx.CTXLOG("Exception %s - retrying? (%d<%d)", e->errorMessage(s).str(), attempts, master->maxRetries);
 
+                attempts++;
```

Comment:
In any case we need to make sure people are aware of this change.  I know I've had to look at this code several times in the past and explain to people how it did behave compared to what they expected.  So there are people out there currently relying on existing behavior.